website/content/api-docs/secret/pki.mdx

```diff
@@ -4099,7 +4099,7 @@ the CRL.
 
 ~> Note: The `unified_crl`, `unified_crl_on_existing_paths`, and
    `cross_cluster_revocation` parameters are all Vault Enterprise only
-   functionality. While they appear in the responses on Vault OSS, they cannot
+   functionality. While they appear in the responses on Vault Community Edition, they cannot
```

Comment:
I notice an inconsistency between "community version" as used in `website/content/api-docs/secret/identity/mfa/index.mdx`, and "Community Edition" as used here elsewhere in other files.

Is the recommendation to prefer the use of "edition" or "version"?